### ISSUE TITLE
Terminal window exiting

### ISSUE THREAD

**Issue by isusanin** [soliton4/nodeMirror#20]
If I issue an "exit" command in the terminal, the terminal windows doesn't close. What's the proper way of closing terminal windows?

**Comment by soliton4** [comment]
right now the terminal windows dont close. you can manualy close it ofc but it will remain in the list of terminals

this is something for 0.2

on unix you can press "restart server" in the config, that will restart the server part of nodeMirror and clear the terminal list

**Comment by isusanin** [comment]
Looking forward to 0.2 then!

**Comment by soliton4** [comment]
me 2 ;D
check out https://github.com/soliton4/promiseland/wiki
http://promise-land.net/

its the result of my experience with nodeMirror and it will be the foundation of 0.2
lots of time goes into this right now

**Comment by isusanin** [comment]
PromiseLand looks interesting. I'm looking forward to learning more about it. Did I understand it correctly that it's a language that compiles to JavaScript and it's main purpose is to deal with the "callback hell"?

**Comment by soliton4** [comment]
@isusanin yes, thats the main motivation behind it, but it lead to other very convenient features integrated on the programming language level.
check out the frames feature
https://github.com/soliton4/promiseland/wiki/frames

a preview of nodemirror 0.2.x is available in the 0.2.x branch.
but dont expect to much jet
